### DISCUSSION TITLE
Update Open In IINA extension on FireFox

### DISCUSSION THREAD

**Discussion by manishbajpai** [iina/iina#4814] [General]
The Chrome ext is updated but FF extension is not updated for 5 years. This leaves key fixes like apostrophe handling broken. Who can update FF ext?

**Comment by low-batt** [comment]
Usually the first step is to describe the problem in detail in an IINA [issue](https://github.com/iina/iina/issues). I searched for such an issue and didn't find one. I could have missed it.

The next step is for a programmer to create a commit with changes to fix the problem in their [fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) and then [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork). IINA is an open source project, so anyone can post a pull request.

After that IINA's senior developers will need to review the proposed changes and merge the changes into the official IINA repository. Always helps if the problem is clearly described and the solution explained.

**Reply by manishbajpai** [reply]
That is not the process for publishing an extension on Firefox. Changes are already in github. The person who owns the extension needs to package and upload. Given that this has not happened for 5+ years, we need to locate the account holder -  "Collider LI" and check if they still have access.

**Reply by low-batt** [reply]
Ah, I was confused. You are referring to [Open In IINA](https://addons.mozilla.org/en-US/firefox/addon/open-in-iina-x/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search) in the [Firefox Browser ADD-ONS](https://addons.mozilla.org/en-US/firefox/) site having last been updated Dec 1, 2018.

Yes, for that we need to hear from @lhc70000. He is very busy at the moment, but I will mention this whenever I can get a hold of him.
